governator-test-spock/src/main/java/com/netflix/governator/test/GovernatorExtension.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.governator.test;
+
+import org.spockframework.runtime.extension.AbstractGlobalExtension;
+import org.spockframework.runtime.model.SpecInfo;
+
+import com.netflix.governator.guice.test.AnnotationBasedTestInjectorManager;
+import com.netflix.governator.guice.test.InjectorCreationMode;
+import com.netflix.governator.guice.test.mocks.MockHandler;
+import com.netflix.governator.test.mock.spock.SpockMockHandler;
+
+import spock.lang.Specification;
+
+public class GovernatorExtension extends AbstractGlobalExtension {
```

Comment:
I'm not familiar with Spock.  Can you add a description about what each of these implementation of Spock API's gives us.